app/notification/model/multi_input_data.py

```diff
@@ -117,3 +119,51 @@ def process_data(cls, data):
                     )
 
         return output
+
+    @classmethod
+    def map_multi_input_data(cls, multi_input_data):
+        """
+        Map the multi-input data to a sorted dictionary and process it.
+
+        Args:
+            multi_input_data (list): The list of dictionaries representing the multi-input data.
+
+        Returns:
+            str: The processed output as a formatted string.
+        """
+        try:
+            key = None
+            value = None
```

Comment:
Are these being used?

Reply:
I dont think so. Thanks for highlighting it 😄